fix: Set the minimum height of the initial project page generated by init to the screen height

SisyphusZheng opened the pull request:
This pull request makes a small enhancement to the `Home` page component in `init/src/init.ts` by ensuring the page takes up the full height of the screen.

* [`init/src/init.ts`](diffhunk://#diff-2d3062a5b277212719aa2542000aa6e3a62e90b7217994970cf4a6dfe01237eaL414-R414): Added the `min-h-screen` class to the outermost `div` to ensure the page has a minimum height equal to the screen height.

<img width="1475" height="600" alt="image" src="https://github.com/user-attachments/assets/84bc0c73-b69c-4e5b-bae6-d5e957fb4491" />

SisyphusZheng committed:
Update init.ts

marvinhagemeister commented on init/src/init.ts:
```diff
@@ -411,7 +411,7 @@ export default define.page(function Home() {
 
   return (
-    <div class="px-4 py-8 mx-auto fresh-gradient">
+    <div class="px-4 py-8 mx-auto fresh-gradient min-h-screen">
```
Nice, can you also add the CSS for that to the fallback code for when people create a project without tailwindcss? The code for that is in the variable `NO_TAILWIND_STYLES`

marvinhagemeister replied:
Ah not quite. The fallback css code is _only_ loaded when you don't pick the "use tailwindcss" option. When you _do_ pick tailwind this will not be used, so it should be a tailwind class name regardless. Tailwind doesn't have a `full-height` class as far as I'm aware.

SisyphusZheng replied:
I finally figured it out. LOL

SisyphusZheng committed:
Add full-height class for improved layout

Introduced a .full-height CSS class with min-height: 100vh and applied it to the main container div to ensure full viewport height coverage, improving vertical alignment and layout consistency.

SisyphusZheng committed:
fmt

SisyphusZheng committed:
Update init.ts

SisyphusZheng committed:
Rename .full-height class to .min-h-screen

The CSS class .full-height was renamed to .min-h-screen for clarity and consistency with naming conventions. No functional changes were made.

SisyphusZheng committed:
Merge branch 'main' into patch-12

marvinhagemeister reviewed (approved):
Sweet, thanks!
